samples/create-batch-prediction-job-video-classification.js

```diff
@@ -0,0 +1,255 @@
+
+    const bigquerySource = inputConfiguration.bigquerySource;
+    console.log(`\t\tBigQuery source`);
+    if (bigquerySource == null) {
+      console.log(`\t\t\tInput uri : {}`);
+    } else {
+      console.log(`\t\t\tInput uri : ${bigquerySource.inputUri}`);
+    }
+
+    const outputConfiguration = response.outputConfig;
+    console.log(`\t\tOutput config`);
+    console.log(
+        `\t\tPredictions format : ${outputConfiguration.predictionsFormat}`,
+    );
+
+    const gcsDestination = outputConfiguration.gcsDestination;
+    console.log(`\t\tGcs destination`);
+    console.log(
+        `\t\t\tOutput uri prefix : ${gcsDestination.outputUriPrefix}`,
+    );
+
+    const bigqueryDestination = outputConfiguration.bigqueryDestination;
+    if (bigqueryDestination == null) {
+      console.log(`\t\tBigquery destination`);
+      console.log(`\t\t\tOutput uri : {}`);
+    } else {
+      console.log(`\t\tBigquery destination`);
+      console.log(`\t\t\tOutput uri : ${bigqueryDestination.outputUri}`);
+    }
+
+    const dedicatedResource = response.dedicatedResource;
+    console.log(`\tDedicated resources`);
+    if (dedicatedResource == null) {
+      console.log(`\t\tStarting replica count : {}`);
+      console.log(`\t\tMax replica count : {}`);
+    } else {
+      console.log(
+          `\ttStarting replica count : \
+          ${dedicatedResource.startingReplicaCount}`,
+      );
+      console.log(
+          `\ttMax replica count : ${dedicatedResource.maxReplicaCount}`,
+      );
+
+      const machineSpec = dedicatedResource.machineSpec;
+      console.log(`\t\tMachine spec`);
+      if (machineSpec == null) {
+        console.log(`\t\t\tMachine type : {}`);
+        console.log(`\t\t\tAccelerator type : {}`);
+        console.log(`\t\t\tAccelerator count : {}`);
+      } else {
+        console.log(`\t\t\tMachine type : ${machineSpec.machineType}`);
+        console.log(`\t\t\tAccelerator type : ${machineSpec.acceleratorType}`);
+        console.log(
+            `\t\t\tAccelerator count : ${machineSpec.acceleratorCount}`,
+        );
+      }
+    }   
+
+    const manualBatchTuningParameters = response.manualBatchTuningParameters;
+    console.log(`\tManual batch tuning parameters`);
+    if (manualBatchTuningParameters == null) {
+      console.log(`\t\tBatch size : {}`);
+    } else {
+      console.log(`\t\tBatch size : ${manualBatchTuningParameters.batchSize}`);
+    }
+
+    const outputInfo = response.outputInfo;
+    if (outputInfo == null) {
+      console.log(`\tOutput info`);
+      console.log(`\t\tGcs output directory : {}`);
+      console.log(`\t\tBigquery output dataset : {}`);
+    } else {
+      console.log(`\tOutput info`);
+      console.log(
+          `\t\tGcs output directory : ${outputInfo.gcsOutputDirectory}`,
+      );
+      console.log(`\t\tBigquery output dataset : 
+            ${outputInfo.bigqueryOutputDataset}`);
+    }
+
+    const error = response.error;
```

Comment:
So this is interesting.  If there are errors, I'd expect the method to throw.  What's up here?